Galway 2020/index.html

```diff
@@ -96,7 +96,56 @@
 
   <!-- Page Content -->
   <div class="container">
+		
+	  <div class="row">
+<div class="col-sm-6 col-md-2 col-lg-2">
+
+<form class="d-flex justify-content-center">
+	<fieldset>
+		<legend>Price Range</legend>
+		
+		
+
+			<div class="panel panel-default ">
+			
+					<div class="panel-body ">
+						<div id="myBtnContainer" class="btn-group-vertical ">
+						  <button class="btn btn-secondary active" onclick="filterSelection('all')"> Show all</button>
+						  <button class="btn btn-secondary" onclick="filterSelection('200')"> &euro;0-&euro;200</button>
+						  <button class="btn btn-secondary" onclick="filterSelection('400')"> &euro;201-&euro;400</button>
+						  <button class="btn btn-secondary" onclick="filterSelection('599')">&euro;401-&euro;600</button>
+						  <button class="btn btn-secondary" onclick="filterSelection('over600')"> over &euro;600</button>
+						</div>
+					</div>
+				</div>
+
+
+	</fieldset>
```

Comment:
Category for different price range is working perfectly, all buttons are clickable